MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 202…

daviesje opened the pull request:
…5.11.18.16.36.32

Other tools:
- conda-build 25.9.0
- rattler-build 0.50.0
- rattler-build-conda-compat 1.4.9

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

daviesje committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.18.16.36.32

Other tools:
- conda-build 25.9.0
- rattler-build 0.50.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ This recipe is using a compiler, which now requires adding a build dependence on `{{ stdlib("c") }}` as well. Note that this rule applies to each output of the recipe using a compiler. For further details, please see https://github.com/conda-forge/conda-forge.github.io/issues/2102.

For **recipe/meta.yaml**:

 * ℹ️ Recipes should usually depend on `astropy-base` as opposed to `astropy`. `astropy-base` only has the required dependancies whereas `astropy` now has all optional dependancies as well.
 * ℹ️ No valid build backend found for Python recipe for package `21cmfast` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.
 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19489479542](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19489479542). Examine the logs at this URL for more detail.</sub>

daviesje committed:
apply linter suggestions, try updating to v4b

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

daviesje committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.18.16.36.32

Other tools:
- conda-build 25.9.0
- rattler-build 0.50.0
- rattler-build-conda-compat 1.4.9

daviesje committed:
remove classy req

daviesje committed:
fix version

daviesje committed:
update hash

daviesje committed:
lower case filenames

daviesje committed:
for debugging: use main github branch

daviesje committed:
install classy (non-conda) and restrict to python >= 3.11

daviesje committed:
restrict all python versions

daviesje committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.21.02.19.18

Other tools:
- conda-build 25.9.0
- rattler-build 0.50.0
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Non noarch packages should have python requirement without any version constraints.
 * ❌ Non noarch packages should have python requirement without any version constraints.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19564368345](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19564368345). Examine the logs at this URL for more detail.</sub>
